### PR TITLE
Release/v1.0.2

### PR DESCRIPTION
**Pull request by SMK1085**

### CODE REVIEW

File: server/lib/service-client/mapping-util.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+};
+
+class MappingUtil {
+  /**
+   * Gets or sets the account company mappings for Madkudu payloads.
+   *
+   * @type {*}
+   * @memberof MappingUtil
+   */
+  clearbitCompanyMapping: Object;
```
Comment (michaloo) [inline server/lib/service-client/mapping-util.js]: clearbit?

File: server/lib/service-client/segment-error.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import type { TSegmentAction } from "../types";
+
+class SegmentError extends Error {
```
Comment (michaloo) [inline server/lib/service-client/segment-error.js]: Can we rename this error? Like `SegmentEndpointError`? or `LogicError` as we prepared it in hull-node?

File: server/lib/service-client/madkudu-error.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/* @flow */
+import type { TMadkuduMethod } from "../types";
+
+class MadkuduError extends Error {
```
Comment (michaloo) [inline server/lib/service-client/madkudu-error.js]: is it TransientError or LogicError?

File: server/lib/service-client/mapping-util.js [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  }
+
+  /**
+   * Maps the input to a segment identify call.
+   *
+   * @param {*} user The hull user.
+   * @returns {*} The segment payload.
+   * @memberof MappingUtil
+   */
+  mapToIdentify(user: any): any {
```
Comment (michaloo) [inline server/lib/service-client/mapping-util.js]: We can use THullUserAttributes here. Do you think we could add Madkudu types to replace those `any` in this file?

File: server/lib/service-client/mapping-util.js (hunk 1 above)
Comment (SMK1085) [reply server/lib/service-client/mapping-util.js]: Yes runs on Clearbit Company Data :-)

Comment (michaloo) [reply server/lib/service-client/mapping-util.js]: okay, so this is most concerning part of this version. I would say it should be a setting with clearbit traits set as defaults in manifest.json - but I know we want to put it quicker, then iterate.

**Pull request merged**